packages/shared/src/graphql/squads.ts

```diff
@@ -155,6 +155,7 @@ export const EDIT_SQUAD_MUTATION = gql`
       type
       description
       image
+      id
```

Comment:
because this was missing, updateSquad was not working correctly

```
    const squads = bootData.squads?.map((bootSquad) =>
      squad.id !== bootSquad.id ? bootSquad : squad,
    );
```